server/bin/pbench-sync-satellite.sh

```diff
@@ -78,6 +78,9 @@ but failed to remove local "${state_change_log}".
 EOF
             pbench-report-status --name ${PROG} --pid ${$} --timestamp $(timestamp) --type error ${index_content}
         fi
+        if [[ -s ${logdir}/mv.log ]] ; then
```

Comment:
Lines 81 thru 83, are not quite right, it seems.  It currently reads as, "if we have a `${logdir}/mv.log` of some non-zero size, remove it."  This means that if the log file is empty, we won't remove it, and so we won't be able to remove the `${logdir}` directory.

Instead, if we change lines 80 thru 83 to be:

```
        else
            # The removal of ${state_change_log} was successful, so we can attempt to remove
            # the ${logdir}mv.log file.
            rm ${logdir}/mv.log
        fi
```